app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -277,6 +280,17 @@ public void onClick(View v) {
     }
 
     @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (requestCode == 1 && resultCode == RESULT_OK) {
```

Comment:
"The integer argument is a "request code" that identifies your request. When you receive the result Intent, the callback provides the same request code so that your app can properly identify the result and determine how to handle it."